docs/docs/how-to/build-shards.md

```diff
@@ -129,18 +157,14 @@ mkdir release
 
 ## Building the Shards Executable ##
 
-We will first build the Debug version of Shards. Navigate to the “debug” folder with the command `cd debug`.
+We will first build the Debug version of Shards. 
 
-```
-cd debug
-```
-
-Input the command `cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug ../..` to generate the build files.
+Input the following command to generate the build files:
 
 === "Command"
 
     ```
-    cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug ../..
+    cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug -B./Debug ..
```

Comment:
We could add a quick "note" to explain the `..` part as people might miss it or not understand why it's there.

It is here because the main `CMakeLists.txt` file is at the repository root level (hence one level up from the `build` folder where we run that command) and that's the file `cmake` looks for to determine what to generate.